tests/committee_tests.py

```diff
@@ -156,13 +169,13 @@ def test_committee_filters(self):
 
         # checking one example from each field
         filter_fields = (
-            # ('committee_id', ['bartlet', 'ritchie']),
-            # ('state', ['CA', 'DC']),
+            ('committee_id', ['bartlet', 'ritchie']),
```

Comment:
I think that the test works fine, but having the committee_id be a name makes it a little confusing for me to read.